virttest/qemu_monitor.py

```diff
@@ -383,9 +392,14 @@ def correct(self, cmd):
         def translate(cmd):
             return "-".join(re.split("[_-]", cmd))
 
+        found = False
         if not self._has_command(cmd):
             for _cmd in self._supported_cmds:
                 if translate(_cmd) == translate(cmd):
+                    found = True
+                elif _cmd == "x-%s" % cmd:
```

Comment:
`elif _cmd == "x-%s" % translate(cmd):` would be more robust.

Reply:
good idea, it should be:  elif  translate(_cmd)  == "x-%s" %  translate(cmd):  return _cmd.  will fix in next update.